app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="h-full bg-gray-300">
+  <div class="flex justify-evenly items-center bg-white shadow-[0_4px_9px_-4px_#3b71ca] h-20">
+    <h2 class="text-center font-serif uppercase"><%= @category.name %></h2>
+    <p>$<%= @category.total_amount %>0 cents</p>
+  </div>
+  <!-- container for all cards -->
+  <div id="card" class="flex flex-col justify-center mt-8 mb-4">
+    <div class="mx-auto">
+      <%= render 'transactions/transaction', collection: @category.transactions %>
+      <% if can? :manage, @category %>
+        <%= link_to "Add a New Transaction", new_category_transaction_path(category_id: @category.id), class: 'actions group relative w-50 flex
+         justify-center py-2 px-4 border border-transparent mt-4 text-sm font-medium
+         rounded-md text-white bg-indigo-600 hover:bg-indigo-700 cursor:pointer focus:outline-none
+         focus:ring-2 focus-2 focus-offset-2 focus:ring-indigo-500 mr-8 ml-8'%>
+      <% end %>
```

Comment:
- [ ] Great job for adding the menu for navigation but it would be great if you added a back arrow as mentioned in the requirements to back to the home page when the user press on it since it is easier for the user so that would improve from user experience 🤩💯🔥 so kindly add it and link it to the home page 💯


![image](https://user-images.githubusercontent.com/58553711/235318139-1c108eed-d774-4dc9-8fd4-1184bfd35401.png)





- Your
![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/52904fff-6ab8-48d6-8eb5-6bffc5a528e0)